.github/workflows/post-review-ci.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   v8-binding-test:
-    runs-on: [self-hosted, linux, freq-scaling-on]
+    runs-on: [self-hosted, linux, freq-scaling-off]
```

Comment:
What happens to our performance tests?
Should we manually check that nothing else is running on the perf runner?